src/angle_finder.cpp

```diff
@@ -0,0 +1,65 @@
+#include <ros/ros.h>
```

Comment:
This node is already is main, maybe if it got out of date? Don' t want to include it again if already in main. Make sure this branch is not "adding" stuff already in main